port/oc_connectivity_internal.h

```diff
@@ -64,6 +64,13 @@ void oc_connectivity_shutdown(size_t device);
  */
 int oc_send_buffer2(oc_message_t *message, bool queue);
 
+ #ifdef OC_DYNAMIC_ALLOCATION 
+/**
+ * @brief send a signal to start consuming network events
+ */
+void oc_connectivity_wakeup(void);
```

Comment:
TODO for me: check if this does not generate a new function for Java bindings

Reply:
Renamed to `oc_network_get_event_queue_length`